ImageGallery/src/org/sleuthkit/autopsy/imagegallery/ImageGalleryController.java

```diff
@@ -739,27 +752,35 @@ public void run() {
                 "(mime_type LIKE '" //NON-NLS
                 + String.join("' OR mime_type LIKE '", FileTypeUtils.getAllSupportedMimeTypes()) //NON-NLS
                 + "') ";
-
-        static final String DRAWABLE_QUERY =
-                //grab files with supported extension
-                "(" + FILE_EXTENSION_CLAUSE
-                //grab files with supported mime-types
-                + " OR " + MIMETYPE_CLAUSE //NON-NLS
-                //grab files with image or video mime-types even if we don't officially support them
-                + " OR mime_type LIKE 'video/%' OR mime_type LIKE 'image/%' )"; //NON-NLS
-
+   
+        final String DRAWABLE_QUERY;
+        final String DATASOURCE_CLAUSE;
+        
         final ImageGalleryController controller;
         final DrawableDB taskDB;
         final SleuthkitCase tskCase;
+        final long dataSourceObjId;
 
         ProgressHandle progressHandle;
-        
         private boolean taskCompletionStatus;
 
-        BulkTransferTask(ImageGalleryController controller, DrawableDB taskDB, SleuthkitCase tskCase) {
+        BulkTransferTask(long dataSourceObjId, ImageGalleryController controller, DrawableDB taskDB, SleuthkitCase tskCase) {
```

Comment:
I wonder if taskDB is too close to tskDB at a quick glance

Reply:
Possibly.  I'll make a note of it and clean it up in a separate cleanup commit as part of EUR-1010.